### PR TITLE
Reset the file select step component on tab change

### PR DESCRIPTION
**Pull request by eldiegod**
Issue: Changing video tabs doesn't reset the file select step #591

### CODE REVIEW

**Comment by eldiegod** [comment]
Found a clean solution for this, I think this should do it

File: src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx [hunk 1]
```diff
@@ -319,6 +319,7 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
           control={control}
           render={({ value }) => (
             <MultiFileSelect
+              key={selectedVideoTab?.id}
```
Comment (drillprop) [inline src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: Is this needed?

**Review by drillprop — APPROVED** [review]
Looks good to me! 👍🏻

**Review by mikkio-j — APPROVED** [review]

File: src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx (hunk 1 above)
Comment (eldiegod) [reply src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: Actually no

**Review by kdembler — APPROVED** [review]
Really neat 👌

**Pull request merged**